webapp/components/User/User.vue

```diff
@@ -26,7 +26,7 @@
         </div>
       </nuxt-link>
     </template>
-    <template slot="popover">
+    <template slot="popover" v-if="showCounts">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-09T17:09:45Z" title="Monday, December 9th 2019, 6:09:45 pm +01:00">Dec 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -26,7 +26,7 @@
         </div>
       </nuxt-link>
     </template>
-    <template slot="popover">
+    <template slot="popover" v-if="showCounts">
```


I would say `showPopover` no?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-09T17:49:58Z" title="Monday, December 9th 2019, 6:49:58 pm +01:00">Dec 9, 2019</time>_

---


yes, that's better